__tests__/pages/index.spec.js

```diff
@@ -3,7 +3,9 @@ import renderer from 'react-test-renderer';
 
 import Index from '../../pages/index';
 
-describe('Index', () => {
+// FIXME: https://github.com/zeit/next.js/issues/3345
```

Comment:
We don't know what's related to this link. Can you add a brief explanation as to what's it's supposed to explain?